framework/assets/yii.js

```diff
@@ -329,7 +329,7 @@ yii = (function ($) {
             }
 
             if (message !== undefined) {
-                pub.confirm(message, function () {
+                $.proxy(pub.confirm, this)(message, function () {
                     pub.handleAction($this, event);
```

Comment:
@lichunqiang I'm taking about your example of `yii.confirm` overriding 2 posts before. Okay, it's fine